salt/journal-cms/cron.sls

```diff
@@ -1,10 +1,10 @@
 #
 # cron jobs
 #
-journal-cms-cron-cache-rebuild:
+journal-cms-cron-drupal:
     cron.present:
-        - identifier: journal-cms-cron-cache-rebuild
```

Comment:
to really remove the previous cron, another `cron.absent` needs to be created (temporarily until is applied to all instances) with the same `name`, `identifier`, and `user`:
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cron.html#salt.states.cron.absent